miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -56,12 +56,19 @@ var BoxPosition = function(opts) {
   };
 
   self.click = function(event) {
-    if (event.ctrlKey) {
+    if (isMultiSelectClick(event)) {
       self.controlClick();
     } else {
       self.normalClick();
     }
   };
+  
+  isMultiSelectClick = function(event) {
+    function isMac() {
+      return navigator.platform.indexOf('Mac') != -1;
+    }
+    return (isMac() && event.metaKey || !isMac && e.ctrlKey);
```

Comment:
I agree that your option is shorter, but I think it sacrifices clarity. But mine also re-creates the `isMac` function each time which is not ideal, so I'll change it and comment for clarity.